docs/manuals/source/DeveloperGuide/release.rst

```diff
@@ -0,0 +1,74 @@
+Releasing Bareos
+----------------
+
+This chapter describes how to release a new version of Bareos.
+The documentation is limited to the changes that need to be done to the sources and the git repository.
+Building and distributing packages is not the scope of this chapter.
+
+.. note::
+   Consider this a step-by-step manual. The sections are meant to be carried out in that order.
+
+Deciding for a version number
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+When you release you will usually need at least two version numbers: the version you're going to release and the version that will follow after that version.
+For details on the numbering-scheme, take a look at :ref:`version-numbers`.
+
+Usually, you will have an existing work-in-progress tag that generates pre-release version numbers for your branch.
+So the version you're going to release is probably the final version for that pre-release.
+
+Preparing the release notes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+For each new release there should be a section in the :ref:`releasenotes`.
+You can use the release notes of a previous release as a template.
+
+The release notes should be committed to the master branch and then cherry-picked to your release branch using ``git cherry-pick -x``.
+
+Update version-dependent files
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+There are a few version-dependent files in the Bareos sources that need to be changed.
+
+version.h
+^^^^^^^^^
+Starting with Bareos :sinceVersion:`19:remove version.h` the values for ``version.h`` are automatically determined by cmake based on git repository data.
```

Comment:
Here I'm not sure. Could you please take a look at the index and see, how this is replaced. I expect as:

- bareos-19
  - remove version.h

Reply:
Yes, that's exactly what is produced in the index.

Reply:
If this is the intention, I've no further objections.